qiskit/ignis/experiments/calibration/methods/cross_resonance.py

```diff
@@ -57,29 +57,43 @@ def __init__(self,
         # something like qubit property table where f01, anharmonicity, T1, T2, etc... exist.
         freq01 = None
 
+        self._name = 'cr_amp'
+        self._qubits = qubits  # todo move to base class
+        self._inst_def = inst_def  # todo move to base class
+        self._calibration_group = calibration_group
+
         u_ch = inst_def.get_channel(qubits, ControlChannel)
 
-        # Create a list of amp parameters on the control channel.
+        # Create a list of amp parameters on the control channel and default to global.
         u_ch_inst = inst_def.get_schedule(cr_name, qubits).filter(channels=[u_ch]).instructions
-        free_names = []
+        self._parameter_names = []
         for instruction in u_ch_inst:
-            free_names.append(instruction[1].name + '.amp')
+            pulse_name, channel, scope_id = instruction[1].name.split('.')
+            p_name = inst_def.pulse_parameter_table.get_full_name('amp', pulse_name, u_ch.name,
+                                                                  scope_id, calibration_group)
+
+            self._parameter_names.append(p_name)
 
         qc = inst_def.get_circuit('xp', (qubits[0], ))
```

Comment:
I think we don't need Xp pulse because this is not CNOT gate. it can show the oscillation with ground state control qubit but just rotation direction is opposite. Because we performs Z measurement the outcome is not sensitive to the direction.

Reply:
Good point. Removed it in ea8d80d